CHANGELOG.md

```diff
@@ -12,8 +12,9 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Removed
 
 ### Fixed
-
+- Mocha Dependency issue in `opensearch-dsl` gem
```

Comment:
Are we adding PR details to Changelogs?

Reply:
Yes. Thanks for catching that. Though I think we should link the GH Issue instead if it's available so we don't have to deal with the chicken-and-egg issue when updating CHANGELOG.md, or when the PR is recreated.